tool/testcoverage

```diff
@@ -26,14 +26,14 @@ case "$ARCH-$OS" in
     *-Darwin)
         CONFIGURATION=Debug
         (
-            cd -- "$CODE"
-            xcrun xcodebuild -config "$CONFIGURATION" clean
+            cd -- "$CODE" &&
+            xcrun xcodebuild -config "$CONFIGURATION" clean &&
```

Comment:
It wasn't immediately clear what this change has to do with shell script checking. Perhaps it's a fix subsequent to running the shell script checker. The PR doesn't appear to mention that fixes are also included.

Reply:
This comment applies in several other places where changes have been made to existing shell scripts

Reply:
> m: It wasn't immediately clear what this change has to do with shell script checking. Perhaps it's a fix subsequent to running the shell script checker. The PR doesn't appear to mention that fixes are also included.

M: It a risk to the MPS if a pull request doesn't fully describe its own contents, so that checkers can't see extraneous stuff that shouldn't be there.  In words, checkers need to be able to check the *completeness* of the change and its description (both directions).  In GitHub, that should be visible in the pull request description.  It could be linked, but must be there.

Reply:
Raise: @thejayps , as part of your proc.review.pi, please consider adding to [entry.universal](https://github.com/Ravenbrook/mps/blob/branch/2023-01-19/review-procedure/procedure/entry.universal.rst) something about a change being completely described somehow.